src/middlewared/middlewared/plugins/cron.py

```diff
@@ -121,4 +121,10 @@ def cron_extend(self, data):
             id
         )
 
+        await self.middleware.call(
+            'service.restart',
+            'cron',
+            {'onetime': False}
```

Comment:
It's `True` by default, so `/etc/rc.d/cron onerestart` won't make sense and we're avoiding that.

Reply:
No, only thing service.restart does is regenetate config file.

Reply:
OK, it regenerates crontab, does crond automatically reread it? And why `{'onetime': False}` is all over the middlewared then? Do you want me to remove this as well?

Reply:
Yes, cron re-reads based on mtime of the file.

Reason it is everywhere is probably because I missed that on reviews (Waqar did those).

No need to change existing, just new ones.